### PR TITLE
S3655: Scaffold rule and UTs

### PR DESCRIPTION
**Pull request by antonioaversa**
Task 2 of https://github.com/SonarSource/sonar-dotnet/issues/6794

Previous task: https://github.com/SonarSource/sonar-dotnet/pull/6844

### CODE REVIEW

File: analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/EmptyNullableValueAccessTest.cs [hunk 1]
```diff
@@ -18,47 +18,68 @@
...
+    public void EmptyNullableValueAccess_Roslyn_CSharp9() =>
+        roslynCS.AddPaths("EmptyNullableValueAccess.CSharp9.cs").WithTopLevelStatements().Verify();
+
+    [TestMethod]
+    public void EmptyNullableValueAccess_Sonar_CSharp10() =>
+        sonar.AddPaths("EmptyNullableValueAccess.CSharp10.cs").WithOptions(ParseOptionsHelper.FromCSharp10).Verify();
+
+    [TestMethod]
+    public void EmptyNullableValueAccess_Roslyn_CSharp10() =>
+        roslynCS.AddPaths("EmptyNullableValueAccess.CSharp10.cs").WithOptions(ParseOptionsHelper.FromCSharp10).Verify();
```
Comment (zsolt-kolbay-sonarsource) [inline analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/EmptyNullableValueAccessTest.cs:74]: Each call should be in a separate lines when chaining methods.

**Review by zsolt-kolbay-sonarsource — APPROVED** [review]
LGTM
Just a comment on formatting.

File: analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/EmptyNullableValueAccessTest.cs (hunk 1 above)
Comment (pavel-mikula-sonarsource) [reply analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/EmptyNullableValueAccessTest.cs:74]: No, this is fine when it fits on a single line without violating the too long line rule.

File: analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RuleChecks/EmptyNullableValueAccessBase.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarAnalyzer.SymbolicExecution.Roslyn.RuleChecks;
+
+public abstract class EmptyNullableValueAccessBase : SymbolicRuleCheck
```
Comment (pavel-mikula-sonarsource) [inline analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RuleChecks/EmptyNullableValueAccessBase.cs:23]: Until we start working on the VB part, this file is not needed and everything can be on the C# side.

**Pull request merged**